src/dvc_data/hashfile/meta.py

```diff
@@ -30,6 +31,8 @@ class Meta:
     inode: Optional[int] = None
     mtime: Optional[float] = None
 
+    remote: Optional[str] = field(default=None, eq=False)
```

Comment:
I am a bit worried about this during `diff`, and we don't support multiple remotes. So I have removed this from being used in comparisons/equality.